src/lib/containers/download_list/CreatePackage.tsx

```diff
@@ -152,14 +152,15 @@ export const CreatePackage = (props: CreatePackageProps) => {
                   ></input>
                   <span className="zip-extension SRC-boldText">.zip</span>
                 </form>
-                <button
-                  className="SRC-standard-button-shape SRC-primary-button action-button"
+                <ReactBoostrap.Button
```

Comment:
No "Before" for this one. We didn't have this component in the styleguide.


https://user-images.githubusercontent.com/17580037/104053710-b9076000-51b9-11eb-8b98-0facb572844a.mov